_source/_posts/2018-10-31-jwts-with-java.md

```diff
@@ -11,7 +11,7 @@ tweets:
 - "Tutorial - JSON Web Tokens with @Java"
 - "Learn how to create JWTs with Java and #JJWT"
 image: blog/featured/okta-java-short-headphones.jpg
-type: conversion
+type: awareness
```

Comment:
@bdemers Are you OK with this change? I believe you helped publish this one.

Reply:
👍